client/services.go

```diff
@@ -60,7 +60,7 @@ type ServiceStatus struct {
 	SubState        string `json:"sub_state"`
 	UnitFileState   string `json:"unit_file_state"`
 	SnapName        string `json:"snap_name"`
-	ServiceName     string `json:"service_name"`
+	AppName         string `json:"service_name"`
```

Comment:
As mentioned in the commit message, I'd like to do this separately to have smaller impact of each change. We will have to look at the REST API to see how many things are affected there.